docker-compose.yml

```diff
@@ -8,6 +8,8 @@ x-defaults: &x-defaults
     POSTGRES_PORT: "${POSTGRES_PORT:-5432}"
     SIGNAL_CLI_REST_API_ENDPOINT: "http://signal:8080"
     SIGNAL_CLI_REST_API_ATTACHMENT_PATH: "signal-cli-config/attachments/"
+    THREE_SIXTY_DIALOG_PARTNER_REST_API_ENDPOINT: "https://hub.360dialog.io/api/v2"
+    THREE_SIXTY_DIALOG_WHATS_APP_REST_API_ENDPOINT: "https://waba.360dialog.io/v1"
```

Comment:
I'll hold off on this change because you have a PR that would consolidate the setup jobs into one and that will affect this suggestion. Maybe you could even add it there?